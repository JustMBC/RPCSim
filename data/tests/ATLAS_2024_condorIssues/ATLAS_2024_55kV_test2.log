000 (47143.000.000) 2025-05-03 23:34:52 Job submitted from host: <[2a05:b400:104:100::500:82f]:19363?addrs=[2a05-b400-104-100--500-82f]-19363&alias=el9-06.hep.phy.cam.ac.uk>
...
040 (47143.000.000) 2025-05-03 23:35:08 Started transferring input files
	Transferring to host: <[2a05:b400:104:200::500:dc03]:45083?addrs=[2a05-b400-104-200--500-dc03]-45083&alias=wcdc-003.hep.phy.cam.ac.uk>
...
040 (47143.000.000) 2025-05-04 02:53:53 Finished transferring output files
...
005 (47143.000.000) 2025-05-04 02:53:53 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 11:42:34, Sys 0 00:00:06  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 11:42:34, Sys 0 00:00:06  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	7339985  -  Run Bytes Sent By Job
	590079228  -  Run Bytes Received By Job
	7339985  -  Total Bytes Sent By Job
	590079228  -  Total Bytes Received By Job
	Partitionable Resources :     Usage  Request Allocated 
	   Cpus                 :      1.99        1         1 
	   Disk (KB)            : 604551     1048576   1049600 
	   GPUs                 :                            0 
	   Memory (MB)          :    278        2048      2048 
	   TimeExecute (s)      :  11920                       
	   TimeSlotBusy (s)     :  11926                       

	Job terminated of its own accord at 2025-05-04T01:53:53Z with exit-code 0.
...
