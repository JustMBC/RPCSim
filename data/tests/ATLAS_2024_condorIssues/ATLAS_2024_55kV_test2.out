Configuration:
   Detector:
	 gap width: 0.1
	 steps: 500
	 electric field: 55000
	 ebar calculation steps: 140
	 anode: 0.12 cm, epsilon=5
	 cathode: 0.12 cm, epsilon=5
	 threshold: 3 pC
   Gas composition:
	 temperature: 293.15
	 pressure: 760
	 gases: 4
	 gas 0 -- name: c2h2f4 - percentage: 64
	 gas 1 -- name: co2 - percentage: 30
	 gas 2 -- name: isobutane - percentage: 5
	 gas 3 -- name: sf6 - percentage: 1
   Simulation parameters:
	 particle: muon
	 momentum: 5e+09
	 x0: 0
	 theta: 0
	 events to simulate: 8
	 threads to use: 4
	 outfile path: out/ATLAS_2024_55kV_test2.dat
	 generator: SFMT
	 random global seed
	 Debug outputs: 0
	 verbose: 1
	 verbosity level: 1
	 snapshots: 0
Init SFMT generator with random seed: 4069176365
MediumMagboltz::SetComposition: C2H2F4/CO2/iC4H10/SF6 (64/30/5/1)
	Gas table file: C2H2F4-0.64_CO2-0.3_iC4H10-0.05_SF6-0.01_temp-293.15_pres-760.gas
MediumMagboltz::LoadGasFile:
    Reading file gastables/C2H2F4-0.64_CO2-0.3_iC4H10-0.05_SF6-0.01_temp-293.15_pres-760.gas.
    Version 13.
    Gas composition set to iC4H10/CO2/SF6/C2H2F4 (5/30/1/64).

Transport parameters:
	alpha: 250.379 cm-1
	eta: 25.7427 cm-1
	E: 55000 kV/cm
	Drift velocity: (0.0130538,0,0) cm/ns
	Ion Drift velocity: (0,0,0) cm/ns
	Diffusion coefficient: (0.00983003, 0.0119947) cm-1/2
	Dt: 0.0153212 ns

EbarTable filename: fDiffT:0.0119947-gapWidth:0.1-eps1:5-eps3:5-cathode:0.12-anode:0.12-Nstep:500-Dx:0.0002-EbarTableSize:140
Hash representation: 3052240547931757472
Computing Ebar Table ... 

TAvalanche::Computing cluster density.
TAvalanche::Computing electron density and cluster size.
Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 0) with Heed track (thread 0) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: 78024121
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 4 electrons produced by heed track.

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 1) with Heed track (thread 1) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: -5440232
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 12 electrons produced by heed track.

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 2) with Heed track (thread 2) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: -628844908
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 16 electrons produced by heed track.

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 3) with Heed track (thread 3) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: -1755576117
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 40 electrons produced by heed track.


2025-05-04.00:58:46 - Avalanche simulation id 1 (0nth simulation) terminated with success (4796 seconds).

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 4) with Heed track (thread 4) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: -628494200
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 2 electrons produced by heed track.


2025-05-04.01:05:00 - Avalanche simulation id 4 (1nth simulation) terminated with success (374 seconds).

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 5) with Heed track (thread 5) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: 1416570971
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 12 electrons produced by heed track.

clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed

2025-05-04.01:20:28 - Avalanche simulation id 0 (2nth simulation) terminated with success (6099 seconds).

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 6) with Heed track (thread 6) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: -2071627122
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 4 electrons produced by heed track.


2025-05-04.01:24:45 - Avalanche simulation id 2 (3nth simulation) terminated with success (6354 seconds).

Longitudinal diffusion generator: SIMD-oriented Fast Mersenne Twister
Initialising avalanche (Id: 7) with Heed track (thread 7) for muon with momentum 5e+09 at position 0 with angle 0
Garfield::RandomEngineSTL seed: 971887584
Cluster density for muon with momentum 5e+09 : 77.9916
TAvalanche1D::initialiseTrackHeed -- 13 electrons produced by heed track.


2025-05-04.01:48:09 - Avalanche simulation id 3 (4nth simulation) terminated with success (7757 seconds).


2025-05-04.02:44:47 - Avalanche simulation id 7 (5nth simulation) terminated with success (4802 seconds).

clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed
clt failed

2025-05-04.02:51:01 - Avalanche simulation id 5 (6nth simulation) terminated with success (6360 seconds).


2025-05-04.02:53:53 - Avalanche simulation id 6 (7nth simulation) terminated with success (5604 seconds).

Failed to load module MathCore
Failed to load module Hist
Failed to load module ROOTDataFrame
Failed to load module SpectrumPainter
Failed to load module ROOTTMVASofieParser
Failed to load module SPlot
Failed to load module TreePlayer
Failed to load module Gpad
Failed to load module Foam
Failed to load module ROOTBrowserv7
Failed to load module RooFitHS3
Failed to load module Physics
Failed to load module ROOTBrowsable
Failed to load module RCsg
Failed to load module Minuit2
Failed to load module Unfold
Failed to load module ROOTHist
Failed to load module PyMVA
Failed to load module Graf3d
Failed to load module FFTW
Failed to load module Quadp
Failed to load module RooFitMore
Failed to load module Hbook
Failed to load module GX11
Failed to load module Postscript
Failed to load module Genetic
Failed to load module FITSIO
Failed to load module HistFactory
Failed to load module ROOTGeomViewer
Failed to load module MathCore
Failed to load module Fumili
Failed to load module FitPanel
Failed to load module Smatrix
Failed to load module Spectrum
Failed to load module Unuran
Failed to load module ROOTFitPanelv7
Failed to load module ASImageGui
Failed to load module Geom
Failed to load module ROOTHistDraw
Failed to load module Graf
Failed to load module Gviz3d
Failed to load module Eve
Failed to load module Minuit
Failed to load module WebGui6
Failed to load module TMVAUtils
Failed to load module RooStats
Failed to load module GeomPainter
Failed to load module GuiHtml
Failed to load module ROOTEve
Failed to load module HistPainter
Failed to load module MLP
Failed to load module Ged
Failed to load module GuiBld
Failed to load module TMVA
Failed to load module TMVAGui
Failed to load module RooFitXRooFit
Failed to load module ROOTWebDisplay
Failed to load module RHTTPSniff
Failed to load module RGL
Failed to load module Tree
Failed to load module GX11TTF
Failed to load module ROOTTMVASofie
Failed to load module EG
Failed to load module X3d
Failed to load module Gui
Failed to load module ASImage
Failed to load module Matrix
Failed to load module Recorder
Failed to load module Gdml
Failed to load module ROOTTreeViewer
Failed to load module MathMore
Failed to load module GenVector
Failed to load module RooFitCore
Failed to load module RooFit
Failed to load module Hist
Failed to load module ROOTNTupleUtil
Failed to load module TreeViewer
Failed to load module MathCore
Failed to load module GenVector
Failed to load module Matrix
Failed to load module Hist
Failed to load module Geom
Failed to load module Gdml
Failed to load module Graf
Failed to load module Gpad
Failed to load module Graf3d
Failed to load module Tree
Failed to load module TreePlayer
Failed to load module ROOTDataFrame
Failed to load module Physics
Failed to load module Postscript
Failed to load module Gui
